src/lib/Type.hs

```diff
@@ -607,6 +628,10 @@ typeCheckPrimCon con = case con of
   ParIndexCon t v -> t|:TyKind >> v|:IdxRepTy >> substM t
   RecordRef xs -> (RawRefTy . StaticRecordTy) <$> traverse typeCheckRef xs
   LabelCon _   -> return $ TC $ LabelType
+  ExplicitDict dictTy _  -> do
+    -- TODO: check method
```

Comment:
Can you please implement that? CI is already picking up errors in the way we used explicit app, so those checks are important